docs/usage/models.md

```diff
@@ -155,7 +155,7 @@ Here a vanilla class is used to demonstrate the principle, but any ORM class cou
 ### Data binding
 
 Arbitrary classes are processed by *pydantic* using the `GetterDict` class (see
-[utils.py](https://github.com/pydantic/pydantic/blob/main/pydantic/utils.py)), which attempts to
+[utils.py](https://github.com/pydantic/pydantic/blob/v1.10.5/pydantic/utils.py)), which attempts to
```

Comment:
Why 
```suggestion
[utils.py](https://github.com/pydantic/pydantic/blob/1.10.X-fixes/pydantic/utils.py)), which attempts to
```

Reply:
@hramezani can you please explain me why? 
Shouldn't the links in docs target the latest stable release?

Reply:
Because then we would have to change all such links on all releases, we could perhaps make that change in future, but that's a bigger issue.

Reply:
Okay, I see. Thanks for the clarification, will commit changes now.